contrib/mesos/pkg/executor/service/service.go

```
@@ -173,6 +173,19 @@ func (s *KubeletExecutorServer) runKubelet(
 		kcfg.DockerDaemonContainer = "" // don't move the docker daemon into a cgroup
 		kcfg.Hostname = kcfg.HostnameOverride
 		kcfg.KubeClient = apiclient
+
+		// taken from KubeletServer#Run(*KubeletConfig)
+		clientConfig, err := s.CreateAPIServerClientConfig()
+		if err == nil {
+			kcfg.KubeClient, err = client.New(clientConfig)
+
+			// make a separate client for events
+			eventClientConfig := *clientConfig
+			eventClientConfig.QPS = s.EventRecordQPS
+			eventClientConfig.Burst = s.EventBurst
+			kcfg.EventClient, err = client.New(&eventClientConfig)
```

don't ignore this error